File: app/src/main/java/ru/vegax/xavier/a3test/models/UserAlbumWrapper.java
package ru.vegax.xavier.a3test.models;

import java.util.List;

public class UserAlbumWrapper {


    private List<UserAlbum> mList;
    private Throwable mThrowable;

    public UserAlbumWrapper(List<UserAlbum> list) {
        mList = list;
    }

    public UserAlbumWrapper(Throwable throwable) {
        mThrowable = throwable;
    }

    public List<UserAlbum> getList() {
        return mList;
    }

    public Throwable getThrowable() {
        return mThrowable;
    }

}

File: app/src/main/java/ru/vegax/xavier/a3test/models/UserAlbumWrapper.kt
package ru.vegax.xavier.a3test.models

class UserAlbumWrapper {


    var list: List<UserAlbum>? = null
    var throwable: Throwable? = null

    constructor(list: List<UserAlbum>?) {
        this.list = list
    }

    constructor(throwable: Throwable) {
        this.throwable = throwable
    }
}


